src/main/webapp/app/shared/components/listing-card/listing-card.tsx

```diff
@@ -0,0 +1,38 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React from 'react';
+import { Row, Col } from 'reactstrap';
+import './listing-card.scss';
+
+export interface IListingCardProps {
+  showActionMenu: boolean;
+  actionMenuHandler?: Function;
+  title?: string;
+}
+
+export class ListingCard extends React.Component<IListingCardProps> {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div className="card-container container">
+        <Row className="align-items-center justify-content-center">
+          <Col xs="10">
+            {this.props.title && this.props.title.hasOwnProperty('value') ? (
+              <span className="font-weight-bold text-dark d-block mb-1">{this.props.title}</span>
+            ) : null}
```

Comment:
```
// will render
true ? <h1>Hello World</h1> : null

// will not render
false ? <h1>Hello World</h1> : null
```
your code is not wrong, but just FYI, there is an easier syntax that can make your life better.

```
// will render
true && <h1>Hello World</h1>

// will not render
false && <h1>Hello World</h1>
```

the resource here explains why
resource: https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

Reply:
just point out another thing here, if you are using the convenient syntax suggested.
Don't fall into the trick of falsy expression. (read the 1st resource until the end, react doc documented it down quite clearly)

so, for your case, 
```
// wrong! The react doc explains why
this.props.title && <h1>Hello World</h1>

// correct
Boolean(this.props.title) && <h1>Hello World</h1>

// wrong! Read the 2nd resource below
new Boolean(this.props.title) && <h1>Hello World</h1>

// correct, but not recommended, I personally think this is less readable. Can we stick to the 1st solution, please?
!!this.props.title && <h1>Hello World</h1>
```
resource: [how to convert values to boolean](https://www.samanthaming.com/tidbits/19-2-ways-to-convert-to-boolean/)

Reply:
Update in f2131ff5e4d6687aed3eb5beb220f1dc9cce08f3